orchard_pallas.py

```diff
@@ -31,6 +31,13 @@ class Fp(FieldElement):
     def from_bytes(buf):
         return Fp(leos2ip(buf), strict=True)
 
+    def random(randbytes):
+        while True:
+            try:
+                return Fp(leos2ip(randbytes(32)), strict=True)
```

Comment:
Use the `Rand` type from `tv_rand.py` instead:
```suggestion
                return Fp(leos2ip(rand.b(32)), strict=True)
```